.github/workflows/pypi-publish.yml

```
@@ -1,7 +1,12 @@
 name: Publish to PyPI
 
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag to publish (for example v2.7.8.post1)"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)